projects/plugins/jetpack/extensions/index.json

```diff
@@ -58,13 +57,7 @@
 		"videopress/video",
 		"videopress/video-chapters"
 	],
-	"experimental": [
-		"ai-image",
-		"ai-paragraph",
-		"anchor-fm",
-		"blogging-prompts",
-		"premium-content"
-	],
+	"experimental": [ "ai-image", "ai-paragraph", "anchor-fm", "blogging-prompt", "premium-content" ],
```

Comment:
Would we consider making it a production block, so self-hosted sites can use it too? It seems like a useful block!

Reply:
I'm open to that--I don't see why it can't be a production block assuming we can get the appropriate jetpack.com doc pages set up.

Reply:
I think that's something Quill can help us with. I created 1200427382410585-as-1204188078771387/f to open a request with the team.